src/context/WalletProvider/config.ts

```diff
@@ -24,12 +24,14 @@ import { NativeTestPhrase } from './NativeWallet/components/NativeTestPhrase'
 import { PortisConnect } from './Portis/components/Connect'
 import { PortisFailure } from './Portis/components/Failure'
 import { PortisSuccess } from './Portis/components/Success'
+import { pairDevice } from './MetaMask/helpers/pairDevice'
 
 export interface SupportedWalletInfo {
   adapter: any
   icon: ComponentWithAs<'svg', IconProps>
   name: string
   routes: RouteProps[]
+  pairDevice?: any
```

Comment:
we should avoid using `any` and use a correct type, which in this case is a function `(props: IPairDeviceProps) => Promise<void>`